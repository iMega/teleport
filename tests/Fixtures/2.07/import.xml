<?xml version="1.0" encoding="UTF-8"?>
<КоммерческаяИнформация xmlns="urn:1C.ru:commerceml_2" xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" ВерсияСхемы="2.07" ДатаФормирования="2015-08-15T16:16:49">
    <Классификатор>
        <Ид>4b86411f-b999-b421-e635-d655ee9fd946</Ид>
        <Наименование>Классификатор (Каталог товаров 4B86411F)</Наименование>
        <Владелец>
            <Ид>53823102-1768-11e5-80cd-6cf049551cd2</Ид>
            <Наименование>Сигма ООО</Наименование>
            <ОфициальноеНаименование>Сигма ООО</ОфициальноеНаименование>
            <ОКПО/>
        </Владелец>
        <Группы>
            <Группа>
                <Ид>438ab755-176a-11e5-80cd-6cf049551cd2</Ид>
                <Наименование>Парфюмерия и косметика</Наименование>
                <Группы>
                    <Группа>
                        <Ид>51e4c212-176a-11e5-80cd-6cf049551cd2</Ид>
                        <Наименование>Парфюмерия</Наименование>
                        <Группы>
                            <Группа>
                                <Ид>58c1e89c-176a-11e5-80cd-6cf049551cd2</Ид>
                                <Наименование>Женская парфюмерия</Наименование>
                            </Группа>
                        </Группы>
                    </Группа>
                </Группы>
            </Группа>
        </Группы>
        <Свойства>
            <Свойство>
                <Ид>bdb690a4-176a-11e5-80cd-6cf049551cd2</Ид>
                <Наименование>Тип аромата</Наименование>
                <ТипЗначений>Справочник</ТипЗначений>
                <ВариантыЗначений>
                    <Справочник>
                        <ИдЗначения>d2a89db4-176a-11e5-80cd-6cf049551cd2</ИдЗначения>
                        <Значение>Цветочный</Значение>
                    </Справочник>
                </ВариантыЗначений>
            </Свойство>
        </Свойства>
    </Классификатор>
    <Каталог СодержитТолькоИзменения="false">
        <Ид>4b86411f-b999-b421-e635-d655ee9fd946</Ид>
        <ИдКлассификатора>4b86411f-b999-b421-e635-d655ee9fd946</ИдКлассификатора>
        <Наименование>Каталог товаров 4B86411F</Наименование>
        <Владелец>
            <Ид>53823102-1768-11e5-80cd-6cf049551cd2</Ид>
            <Наименование>Сигма ООО</Наименование>
            <ОфициальноеНаименование>Сигма ООО</ОфициальноеНаименование>
            <ОКПО/>
        </Владелец>
        <Товары>
            <Товар>
                <Ид>faf9ca10-176a-11e5-80cd-6cf049551cd2</Ид>
                <Артикул>GU641DWBZ539</Артикул>
                <Наименование>Туалетная вода Gucci Flora Eau Fraiche</Наименование>
                <БазоваяЕдиница Код="796" НаименованиеПолное="Штука" МеждународноеСокращение="PCE">
                    <Пересчет>
                        <Единица>796</Единица>
                        <Коэффициент>1</Коэффициент>
                    </Пересчет>
                </БазоваяЕдиница>
                <Группы>
                    <Ид>58c1e89c-176a-11e5-80cd-6cf049551cd2</Ид>
                </Группы>
                <Описание>Яркая и сверкающая туалетная вода Flora by Gucci Eau Fraiche прекрасно подойдет для молодой, романтичной и кокетливой девушки, которой свойственна некоторая спонтанность. Легкость и игривость композиции основана на гармоничном сочетании нежных цветочных и фруктовых нот, дополненных чувственностью и теплом розового перца, сандала и пачули. В этом аромате смешаны задорное и радостное настроение, веселая атмосфера праздника и романтическая летняя нотка.</Описание>
                <Картинка>import_files/fa/faf9ca10176a11e580cd6cf049551cd2_1aa66e59176b11e580cd6cf049551cd2.jpg</Картинка>
                <Картинка>import_files/fa/faf9ca10176a11e580cd6cf049551cd2_1aa66e5a176b11e580cd6cf049551cd2.jpg</Картинка>
                <Картинка>import_files/fa/faf9ca10176a11e580cd6cf049551cd2_1aa66e5b176b11e580cd6cf049551cd2.jpg</Картинка>
                <Изготовитель>
                    <Ид>8ff25e36-4349-11e5-80d0-6cf049551cd2</Ид>
                    <Наименование>Gucci</Наименование>
                </Изготовитель>
                <ЗначенияСвойств>
                    <ЗначенияСвойства>
                        <Ид>bdb690a4-176a-11e5-80cd-6cf049551cd2</Ид>
                        <Значение>d2a89db4-176a-11e5-80cd-6cf049551cd2</Значение>
                    </ЗначенияСвойства>
                </ЗначенияСвойств>
                <СтавкиНалогов>
                    <СтавкаНалога>
                        <Наименование>НДС</Наименование>
                        <Ставка>18</Ставка>
                    </СтавкаНалога>
                </СтавкиНалогов>
                <ЗначенияРеквизитов>
                    <ЗначениеРеквизита>
                        <Наименование>ВидНоменклатуры</Наименование>
                        <Значение>Туалетная вода</Значение>
                    </ЗначениеРеквизита>
                    <ЗначениеРеквизита>
                        <Наименование>ТипНоменклатуры</Наименование>
                        <Значение>Товар</Значение>
                    </ЗначениеРеквизита>
                    <ЗначениеРеквизита>
                        <Наименование>Полное наименование</Наименование>
                        <Значение>Туалетная вода Gucci Flora Eau Fraiche</Значение>
                    </ЗначениеРеквизита>
                </ЗначенияРеквизитов>
            </Товар>
            <Товар>
                <Ид>c5211d29-197f-11e5-80cd-6cf049551cd2</Ид>
                <Артикул/>
                <Наименование>Монтаж сети</Наименование>
                <БазоваяЕдиница Код="796" НаименованиеПолное="Штука" МеждународноеСокращение="PCE">
                    <Пересчет>
                        <Единица>796</Единица>
                        <Коэффициент>1</Коэффициент>
                    </Пересчет>
                </БазоваяЕдиница>
                <Описание/>
                <СтавкиНалогов>
                    <СтавкаНалога>
                        <Наименование>НДС</Наименование>
                        <Ставка>18</Ставка>
                    </СтавкаНалога>
                </СтавкиНалогов>
                <ЗначенияРеквизитов>
                    <ЗначениеРеквизита>
                        <Наименование>ВидНоменклатуры</Наименование>
                        <Значение>Услуга</Значение>
                    </ЗначениеРеквизита>
                    <ЗначениеРеквизита>
                        <Наименование>ТипНоменклатуры</Наименование>
                        <Значение>Услуга</Значение>
                    </ЗначениеРеквизита>
                    <ЗначениеРеквизита>
                        <Наименование>Полное наименование</Наименование>
                        <Значение>Монтаж сети</Значение>
                    </ЗначениеРеквизита>
                </ЗначенияРеквизитов>
            </Товар>
        </Товары>
    </Каталог>
</КоммерческаяИнформация>
