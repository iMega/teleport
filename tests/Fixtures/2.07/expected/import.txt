{"entityType":40,"changes":true}
{"entityType":1,"guid":"438ab755-176a-11e5-80cd-6cf049551cd2","parent":"","title":"Парфюмерия и косметика","slug":"parfumeriya-i-kosmetika"}
{"entityType":1,"guid":"51e4c212-176a-11e5-80cd-6cf049551cd2","parent":"438ab755-176a-11e5-80cd-6cf049551cd2","title":"Парфюмерия","slug":"parfumeriya"}
{"entityType":1,"guid":"58c1e89c-176a-11e5-80cd-6cf049551cd2","parent":"51e4c212-176a-11e5-80cd-6cf049551cd2","title":"Женская парфюмерия","slug":"zhenskaya-parfumeriya"}
{"entityType":10,"guid":"bdb690a4-176a-11e5-80cd-6cf049551cd2","title":"Тип аромата","slug":"tip-aromata","val_type":"select","parent_guid":""}
{"entityType":10,"guid":"d2a89db4-176a-11e5-80cd-6cf049551cd2","title":"Цветочный","slug":"cvetochnyy","val_type":"","parent_guid":"bdb690a4-176a-11e5-80cd-6cf049551cd2"}
{"entityType":20,"title":"Туалетная вода Gucci Flora Eau Fraiche","descr":"Яркая и сверкающая туалетная вода Flora by Gucci Eau Fraiche прекрасно подойдет для молодой, романтичной и кокетливой девушки, которой свойственна некоторая спонтанность. Легкость и игривость композиции основана на гармоничном сочетании нежных цветочных и фруктовых нот, дополненных чувственностью и теплом розового перца, сандала и пачули. В этом аромате смешаны задорное и радостное настроение, веселая атмосфера праздника и романтическая летняя нотка.","guid":"faf9ca10-176a-11e5-80cd-6cf049551cd2","slug":"tualetnaya-voda-gucci-flora-eau-fraiche","catalog_guid":"4b86411f-b999-b421-e635-d655ee9fd946","article":"","img":"","img_prop":""}
{"entityType":30,"type":"prop","guid":"faf9ca10-176a-11e5-80cd-6cf049551cd2","label":"bdb690a4-176a-11e5-80cd-6cf049551cd2","val":"d2a89db4-176a-11e5-80cd-6cf049551cd2","labelSlug":"","countAttr":"","valSlug":"d2a89db4-176a-11e5-80cd-6cf049551cd2","_visible":0}
{"entityType":30,"type":"attr","guid":"faf9ca10-176a-11e5-80cd-6cf049551cd2","label":"ВидНоменклатуры","val":"Туалетная вода","labelSlug":"vidnomenklatury","countAttr":3,"valSlug":"tualetnaya-voda","_visible":0}
{"entityType":30,"type":"attr","guid":"faf9ca10-176a-11e5-80cd-6cf049551cd2","label":"ТипНоменклатуры","val":"Товар","labelSlug":"tipnomenklatury","countAttr":3,"valSlug":"tovar","_visible":0}
{"entityType":30,"type":"attr","guid":"faf9ca10-176a-11e5-80cd-6cf049551cd2","label":"Полное наименование","val":"Туалетная вода Gucci Flora Eau Fraiche","labelSlug":"polnoe-naimenovanie","countAttr":3,"valSlug":"tualetnaya-voda-gucci-flora-eau-fraiche","_visible":0}
{"entityType":20,"title":"Монтаж сети","descr":"","guid":"c5211d29-197f-11e5-80cd-6cf049551cd2","slug":"montazh-seti","catalog_guid":"4b86411f-b999-b421-e635-d655ee9fd946","article":"","img":"","img_prop":""}
{"entityType":30,"type":"attr","guid":"c5211d29-197f-11e5-80cd-6cf049551cd2","label":"ВидНоменклатуры","val":"Услуга","labelSlug":"vidnomenklatury","countAttr":3,"valSlug":"usluga","_visible":0}
{"entityType":30,"type":"attr","guid":"c5211d29-197f-11e5-80cd-6cf049551cd2","label":"ТипНоменклатуры","val":"Услуга","labelSlug":"tipnomenklatury","countAttr":3,"valSlug":"usluga","_visible":0}
{"entityType":30,"type":"attr","guid":"c5211d29-197f-11e5-80cd-6cf049551cd2","label":"Полное наименование","val":"Монтаж сети","labelSlug":"polnoe-naimenovanie","countAttr":3,"valSlug":"montazh-seti","_visible":0}
